packages/angular/cli/src/utilities/environment-options.ts

```diff
@@ -14,9 +14,23 @@ function isPresent(variable: string | undefined): variable is string {
   return typeof variable === 'string' && variable !== '';
 }
 
+function isEnabled(variable: string): boolean {
+  return variable === '1' || variable.toLowerCase() === 'true';
```

Comment:
This means that `VAR=2` or `VAR=test` would be considered falsey, is that the behavior we want? I imagine it could trip some people up.

I wonder if it might more intuitive to define `isDisabled()` as `variable === '0' || variable === '' || variable.toLowerCase() === 'false'` and then make `isEnabled()` negate it? That way it would follow the truthiness semantics of JS and other languages.

Alternatively it might be a better idea to actually limit allowed inputs to `true` / `false` / `0` / `1` and hard error on anything else. That probably requires more documentation, but might give a cleaner abstraction.

Reply:
This is actually the same code used in other devkit packages. So if we want to change this, I think we should re-evaluate all the packages and do it in a seperate PR.

Reply:
Hmm, I agree we should be consistent at least. At minimum they should probably share this code so we don't accidentally decouple them. Let's leave it for now and we can reevalute if we want to move forward with changing it after this PR.